pages/viewPlans/index.js

```diff
@@ -32,26 +27,33 @@ const StyledList = styled.ul`
   }
 `;
 
+const ButtonContainer = styled.div`
+  width: 100%;
+  text-align: center;
+  margin-top: 2rem;
+`;
+
 export default function ViewAllPlans() {
-  const { data } = useAllPlans();
+  const { data, isLoading } = useAllPlans();
 
-  if (!data) return <Loading />;
+  if (isLoading || !data) return <Loading />;
 
   return (
     <OverviewContainer>
-      <StyledParagraph>
-        {data.length === 1
-          ? `There is ${data.length} template in your vault`
-          : `There are ${data.length} templates in your vault`}
-      </StyledParagraph>
+      <TemplateCount>
+        Found {data.length} {data.length === 1 ? `template` : `templates`}
+      </TemplateCount>
+
       <StyledList>
```

Comment:
What if the length of `data` is 0? Then we would end up with an empty `<ul></ul>`. 

We could guard against that with the following code. 

``` js
{data.length > 0 && (
  <StyledList>
    {data.map((plan) => (
      <li key={plan._id}>
        <TemplateCard templateData={plan} />
      </li>
    ))}
  </StyledList>
)}
```